frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -239,7 +239,7 @@ const ManagePolicyPage = ({
       return globalPoliciesAPI.loadAllNew(queryKey[0]);
     },
     {
-      enabled: isRouteOk,
+      enabled: isRouteOk && !isAnyTeamSelected,
```

Comment:
What happens if you browse to a team page first? Wouldn't the inherited policies table be empty in that case?

Another possibility would be to revert this and use adjust the stale time as a way to reduce server demands while still keeping the inherited policies table relatively fresh.

Reply:
Will read and give this thought after my appointment + sprint demo, thanks for the solid reviews!

Reply:
@gillespi314 The inherited policies are now being returned with the team policies API as another object with an array of global policies <3